ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -192,11 +189,41 @@ public void execute(
       throw new KsqlException("Cannot insert values into windowed stream/table!");
     }
 
+    final ReservedInternalTopics internalTopics = new ReservedInternalTopics(ksqlConfig);
```

Comment:
I just created the internal topics in the `DistributorExecutor` constructor for now. The `InsertValuesExecutor` and `ListTopicsExecutor` will require more changes in several files to pass the `KsqlConfig`. I'd like to create a follow-up PR for that change only to avoid making this PR bigger. is that ok?

Reply:
Don't mind. I don't think this PR is too large yet; adding a few more files where a param is injected and passed through won't, IMHO, make this PR unwieldy. Up to you.  

On the flip side, I don't think we should merge this with the change in how you create the `ProcessingLoggerConfig`, which may necessitate passing more things around anyway. 

IMHO, wiring this up is part of this change. But I'll leave it up to you.